client/server/bundler/assets-writer.js

```diff
@@ -49,16 +51,13 @@ Object.assign( AssetsWriter.prototype, {
 
 			for ( const name in stats.assetsByChunkName ) {
 				// make the manifest inlineable
-				if ( String( name ).startsWith( 'manifest' ) ) {
-					// Usually there's only one asset per chunk, but when we build with sourcemaps, we'll have two.
-					// Remove the sourcemap from the list and just take the js asset
-					// This may not hold true for all chunks, but it does for the manifest.
-					const jsAsset = _.head(
-						_.reject( _.castArray( stats.assetsByChunkName[ name ] ), ( asset ) =>
-							_.endsWith( asset, '.map' )
-						)
-					);
-					statsToOutput.manifests[ name ] = compilation.assets[ jsAsset ].source();
+				if ( String( name ).startsWith( this.options.runtimeChunk ) ) {
+					// Runtime chunk will have two files due ExtractManifestPlugin. Both need to be inlined.
```

Comment:
This doesn't seem to be true. Maybe it was true at some point, but now, the `runtime` chunk is loaded from a URL (not inlined), while only the `manifest` chunk is inlined.

Reply:
After the last change both are inlined. I think we can tackle moving `runtime` to an external script (if we are confident that the hash will change when the css map will change) in a different PR, to be safer.